src/remote_connector_dao/SignalCounter.py

```diff
@@ -0,0 +1,40 @@
+class SignalCounter:
+    def __init__(self) -> None:
+        self.elapsed_time = []
+        self.coincidences = []
+        self.channel1 = []
+        self.channel2 = []
+
+    def record_measurement(
+        self,
+        channel1_instant_signal,
+        channel2_instant_signal,
+        coincidences_instant_signal,
+    ):
+        previous_elapsed_time = self.elapsed_time[-1] if self.elapsed_time else 0
+        self.channel1.append(channel1_instant_signal)
+        self.channel2.append(channel2_instant_signal)
+        self.coincidences.append(coincidences_instant_signal)
+        self.elapsed_time.append(previous_elapsed_time + 1)
+
+        if len(self.elapsed_time) > 50:
+            self.channel1.pop(0)
+            self.channel2.pop(0)
+            self.coincidences.pop(0)
+            self.elapsed_time.pop(0)
+
+    @property
+    def single_channels_max_value(self):
+        return max(self.channel1.extend(self.channel2), default=0)
```

Comment:
Ixii, isso aqui vai te causar um erro grave!!
o `extend` altera a lista existente, ele não cria uma nova. É o que chamamos the "in place". Assim, cada vez que vc chamar `single_channels_max_value` a lista `channel1` irá receber os items da lista `channel2`.

Veja o exemplo abaixo. Para gerar uma lista nova, vc pode utilizar o operador `+`
![Screen Shot 2023-09-29 at 18 30 23](https://github.com/borgesgfj/photon_counter/assets/4041238/73ae1877-15f3-4ae3-a8c2-7577fab2fe98)